ios/Classes/FlutterPDFView.m

```diff
@@ -96,35 +94,7 @@ - (instancetype)initWithFrame:(CGRect)frame
                     [_pdfView.document unlockWithPassword:password];
                 }
 
-                NSUInteger pageCount = [document pageCount];
-            
-                if (pageCount <= defaultPage) {
-                    defaultPage = pageCount - 1;
-                }
-
-                PDFPage* page = [document pageAtIndex: defaultPage];
-                [_pdfView goToPage: page];
```

Comment:
please restore the default page logic

Reply:
@endigo Done! Thanks for the prompt feedback!